### PR TITLE
Dashboard view

### LINKED ISSUE #8: Dashboard view
## Value statement 
As a user
I need an dashboard view
so that I get an overview of expenses and revenues (related to each other) of the team 

## Description 

image

## Acceptance criteria
- The dashboard shows one card with the title "Gewinn&Verlust"
- The card holds three main values: 
  - sum of all expenses
  - sum of all revenues
  - the result of revenues - expenses 
- just basic styling! (without the bars showed in the screenshot)

## Tasks
- [ ] create the three values in the dashboard page and 
- [ ] import the data of expenses and revenues using the existing api routes
- [ ] show the result

## Size
medium

### PR DESCRIPTION
**Pull request by edanum**
fixes #8

### CODE REVIEW

File: components/card.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  justify-content: center;
+  align-items: flex-start;
+  padding: 15px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 9px;
+  width: 100%;
+
+  &:hover {
+    cursor: pointer;
+  }
```
Comment (Kheirah) [inline components/card.js]: Do you have a feature in mind which you plan to implement in future stories concerning the hover effect of the card?
Currently, it's confusing to have a pointer cursor but not be able to click the card. I mean, I can click it, but it's not leading me anywhere and nothing else is popping up as a response to my thinking I could interact with the card.

File: components/expenses/expenseDetail.js [hunk 2]
```diff
@@ -1,70 +1,57 @@
...
-              alt="receipt image"
-            />
-          </Receipt>
-        )}
+    <Card>
+      <CardHeader>#{expense.receiptNumber}</CardHeader>
+      <Amount>{expense.amount.toFixed(2).replace(".", ",")}€</Amount>
+      <Description>{expense.description}</Description>
+      {expense.receipt === "" ? (
+        ""
```
Comment (Kheirah) [inline components/expenses/expenseDetail.js]: Conventionally, you would use `null` when you want to display nothing in jsx.

File: pages/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  if (!expenseData || !revenueData) return <div ref={container}></div>;
+  //
+
+  //GENERATE FINANCE DATE
+  function getSum(array) {
+    let sum = 0;
+    const amounts = array.map((item) => item.amount);
+
+    amounts.forEach((amount) => (sum = sum + amount));
+    return sum;
```
Comment (Kheirah) [inline pages/dashboard.js]: Mhh, I can smell a `reduce` opportunity here!
Your code works 100%. But if you like a challenge, try your hand at writing this with array's reduce function!

**Review by Kheirah — APPROVED** [review]
Looking good!

Just have a look at the comments and you're good to go!

File: pages/dashboard.js (hunk 3 above)
Comment (edanum) [reply pages/dashboard.js]: challenge accepted! Thank you for the hint :)

**Pull request merged**